### DISCUSSION TITLE
Opening old Pika Backup archive on a new PC

### DISCUSSION THREAD

**Discussion by Richie87G** [community/community#144870] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I built a new computer and have the Pika Backup Archibe from the previous machine.  I'd like to restore the archive contents to the new machine but in PIKA I don't see a way to open an old repository, I only see a way to create a NEW repository. 
If I try to point Pika at the old archive it says:

FAILED TO INITIALIZE REPOSITORY
A REPOSITORY ALREADY EXISTS AT THIS LOCATION.

I don't see any way to open an old archive.

Any help is greatly appreciated.
Thanks all,
Rich

**Comment by SamuelYiuSY** [comment]
Upon pressing the + sign, you should be able to see a group of buttons under the label "Use Existing Repository" below the "Create new repository" group of buttons. Can you see that in your Pika Backup?

**Reply by Richie87G** [reply]
Ah! On my screen it didn't look like scrolling would have revealed anything but scrolling down I do indeed see use existing repository as an option. Thank you so much.
